server/mergin/sync/storages/disk.py

```diff
@@ -112,6 +113,13 @@ def __init__(self, project):
         self.project_dir = self._project_dir()
         self.geodiff = GeoDiff()
         self.gediff_log = io.StringIO()
+        self.geodiff_working_dir = os.path.abspath(
```

Comment:
Just wondering about CE/EE here. Do we want to use this functionality on the background in every instance? We can test If GEODIFF_WORKING_DIR exists and If not -> do not copy data somewhere to temp dir on the local images. It could be source of issues.

Reply:
It is implemented in the way it will work for all environments. Even on single machine host and local volume. I do not think it is a big deal to leave it there (no much overhead) and in fact I would favor working dir and storage dir separated in all cases.

Reply:
question here https://github.com/MerginMaps/server/pull/278#discussion_r1749895696